client/src/components/adminPage/stats/Stats.tsx

```diff
@@ -0,0 +1,37 @@
+import { Card, CardActionArea } from '@mui/material';
+import { UsersSignedUp } from './UsersSignedUp';
+import { WorkoutsInSystem } from './WorkoutsInSystem';
+
+export function Stats() {
+    return (
+        <div className="stats-section">
```

Comment:
I wouldn't use div here but since its an admin page it doesnt really matter :), Try container instead